runtime/src/lib.rs

```diff
@@ -558,7 +570,7 @@ parameter_types! {
 }
 
 /// No local origins on this chain are allowed to dispatch XCM sends/executions.
-pub type LocalOriginToLocation = SignedToAccountId32<Origin, AccountId, RelayNetwork>;
+pub type LocalOriginToLocation = ();
```

Comment:
I guess this is something that we'll need to change for `canvas`.

Reply:
Yeah, we'll need to definitely revisit our XCM settings after we a) learn more about what they actually do and b) determine how we actually want to use XCM in the context of Canvas